features/support/journey_helpers.rb

```diff
@@ -61,12 +74,27 @@ def select_random_upper_tier_options(carrier)
 end
 
 def select_random_lower_tier_options
+  return answer_random_lower_tier_questions unless @resource_object == :new_registration
+
+  i = rand(2)
+  if i == 0
+    @journey.check_your_tier_page.submit(option: :unknown)
+
+    answer_random_lower_tier_questions
+
+    @journey.standard_page.submit
```

Comment:
Can we add a comment here to say which page it's on to make it easier to understand the test?
```suggestion
    # Proceed from "You need to register as a lower tier waste carrier":
    @journey.standard_page.submit
```